.github/workflows/go-auto-approve.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: setup go
         uses: actions/setup-go@v1
         with:
-          go-version: '1.13.8'
+          go-version: '1.14.2'
```

Comment:
I don't know how much it matters, but I left this at `1.14` in the `milmove` repository since it appeared to me we were still on `1.14` for the overall project based on the docker image.  Seems like the same case here?

Reply:
Ahh, that makes sense. We have a new docker image that probably hasn't been pushed to both repos. I'll try to do that later today so everything is on 1.14.2.